src/Runner.Worker/ExecutionContext.cs

```diff
@@ -365,6 +366,10 @@ public IExecutionContext CreateChild(
 
             var child = new ExecutionContext(this, isEmbedded);
             child.Initialize(HostContext);
+            if ((Global.Variables.GetBoolean("RunService.FixEmbeddedIssues") ?? false) && embeddedIssueCollector != null)
+            {
+                child._embeddedIssueCollector = embeddedIssueCollector;
```

Comment:
Currently `_embeddedIssueCollector` is incorrectly set in the [constructor here](https://github.com/actions/runner/pull/3446/files#diff-4085092f16aac893358021b20c7a92b02a504a1a99830bc88454ae0380638c45R182)